erpnext/projects/doctype/task/task.py

```diff
@@ -43,6 +44,12 @@ def validate_dates(self):
 		if self.act_start_date and self.act_end_date and getdate(self.act_start_date) > getdate(self.act_end_date):
 			frappe.throw(_("'Actual Start Date' can not be greater than 'Actual End Date'"))
 
+		if(self.project):
+			if frappe.db.exists("Project", self.project):
+				doc = frappe.get_doc("Project", self.project)
```

Comment:
Avoid using `get_doc` for getting only a field's value, use `frappe.db.get_value` instead.

Reply:
Done